CI/macos/02_build_obs.sh

```diff
@@ -69,6 +69,14 @@ _configure_obs() {
         UNITTEST_OPTIONS="-DENABLE_UNIT_TESTS=ON"
     fi
 
+    if [ "${CI}" ]; then
+        if [ "${ARCH}" = "x86_64" ]; then
+            BUILD_NUMBER=${GITHUB_RUN_ID}
+        else
+            BUILD_NUMBER=$((${GITHUB_RUN_ID} + 1))
```

Comment:
This exists as a workaround for Sparkle only allowing one item per "version" in the appcast, even if they are on different channels.

Sadly the only real solution to this would be to continue having separate arm64/x86 appcasts, which I may go back to tomorrow for the sake of fixing this.

Reply:
That's annoying, maybe report this issue upstream, as having arm64 and x86_64 items for the same version+build seems like a very obvious use case for any application that has to ship an entire Google browser with itself..

Reply:
I opened a discussion for now to see what they think: https://github.com/sparkle-project/Sparkle/discussions/2283